### PR TITLE
Add link to RSS feed in blog header

### LINKED ISSUE #21: No link to RSS feed in template
It still exists and is referenced properly in the HTML head but I don't see it linked anywhere in the main template

### PR DESCRIPTION
**Pull request by rmcintosh**
Closes #21 

screen shot 2017-01-22 at 11 40 46 am

### CODE REVIEW

**Comment by eatonphil** [comment]
Hmm, I think I'd prefer to see it as a white icon to the right of the "Engineering" title or at the bottom of the page without an icon just saying RSS Feed beneath the linode.com and Developers links.

**Comment by rmcintosh** [comment]
I like the white icon idea a lot (I'm partial to an icon for RSS vs. text for no particular reason) -- how's this look?

screen shot 2017-01-22 at 12 14 45 pm

**Comment by rmcintosh** [comment]
I should probably link to the feed, huh?

**Comment by eatonphil** [comment]
Nice. Could you try making the padding on the left of the icon the same as the left padding on the Engineer title?

**Comment by rmcintosh** [comment]
I sort of feel like it looks a little nicer nestled against the "Engineering" text but I do not feel strongly about it at all

screen shot 2017-01-22 at 12 44 37 pm

File: css/blog.scss [hunk 1]
```diff
@@ -41,6 +41,14 @@ h3 {
     }
 }
 
+.fa-rss {
+    color: white;   
+}
+
+.header-rss {
+    padding-left: 5px;
+}
```
Comment (eatonphil) [inline css/blog.scss]: You can follow the sass  below and put this right about line 59 in this commit nested in the .header section. Something like this:

```
    &-rss {
        padding-left: 5px;
    }

    &-brand {
    ....
```

Comment (rmcintosh) [inline css/blog.scss]: ah ... the fact that .header-* did it this way should've clued me in...

**Comment by eatonphil** [comment]
Cool, thanks!

**Pull request merged**